experanto/datasets.py

```diff
@@ -254,7 +265,12 @@ def __init__(
         root_folder: str,
         global_sampling_rate: None,
         global_chunk_size: None,
+        add_behavior_as_channels: bool = False,
+        replace_nans_with_means: bool = False,
+        cache_data: bool = False,
+        out_keys: Optional[Iterable] = None,
```

Comment:
for future - maybe add `in_keys` as well, as we might want to load only video and responses but not behaviour and that might save quite a bit of cpu